app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,272 @@
+  <string name="title_activity_my_data">Mijn gegevens</string>
+  <string name="CC0">CC0 - zonder naamsvermelding</string>
+  <string name="CC4">CC4. 0 met naamsvermelding (wordt niet meer ondersteund)</string>
+  <string name="tvLicense">Fotolicentie</string>
+  <string name="tvOwnPhoto">Zijn de foto\'s die je indient door jou genomen?</string>
+  <string name="rbOwnPhotoYes">Ja</string>
+  <string name="rbOwnPhotoNo">Nee</string>
+  <string name="tvLinking">Wil je vermeld worden in onze fotodatabank?</string>
+  <string name="etLinking">Voer de url in van je account</string>
+  <string name="rbLinkingYes">Ja</string>
+  <string name="rbLinkingNo">Nee (wordt anoniem opgeslagen)</string>
+  <string name="rbLinkingwebpage">Eigen website</string>
+  <string name="tvNickname">Bijnaam</string>
+  <string name="menu_title_your_own_station_photos">Mijn foto\'s</string>
+  <string name="rbLinkingWith">Linken met: </string>
+  <string name="ownPhoto">eigen foto</string>
+  <string name="bt_mydata_commit">Opslaan</string>
+  <string name="bt_mydata_clear">Wissen</string>
+  <string name="preferences_saved">Je gegevens zijn opgeslagen. </string>
+  <string name="preferences_cleared">Je gegevens zijn gewist. Zodra je de pagina heropent kun je ze opnieuw invoeren.</string>
+  <string name="tvOwnPhotos">Zijn dit je eigen foto\'s?</string>
+  <string name="nav_oepnv">met bus en train</string>
+  <string name="nav_car">met de car</string>
+  <string name="nav_bike">met de bike</string>
+  <string name="nav_walk">wandelen</string>
+  <string name="nav_show">alleen de locatie</string>
+  <string name="navMethod">Route tonen</string>
+  <string name="title_activity_gallery">Mijn stationsfoto\'s</string>
+  <string name="activitynotfound">Je hebt geen navigatie-app op je telefoon.</string>
+  <string name="count_of_images">Je hebt %1$d stationsfoto(\'s)</string>
+  <string name="search_hint">Station zoeken</string>
+  <string name="app_info_title">Over de app</string>
+  <string name="app_info_ok">Oké</string>
+  <string name="app_info_text">App-ontwikkeling: \n<b><a href="http://www.gaby-becker.de">Gaby Becker</a></b>
+        \nin 08/2016 tijdens het opensourceproject van <b><a href="http://www.deutschlands-bahnhoefe.org">Deutschlands Bahnhöfe</a></b>.          \n\nDe stationsinformatie is aangeleverd door Deutsche Bahn in de context van het lokale <b><a href="http://data.deutschebahn.com/pages/nutzungsbedingungen">OpenData-project</a></b>.          \n\nHet json-bestand dat de foto\'s en stationsinformatie samenvoegt is aangeleverd door de Java-app, ontwikkeld door <b><a href="https://twitter.com/storchp">Peter Storch</a></b>.           \n\nDe app gebruikt de <a href="https://design.google.com/icons/">material-design-icons van Google,</a> uitgebracht onder de Apache-licentie, versie 2.0\n\n<b>Medewerkers</b> van het 'Deutschlands-BahnhÃ¶fe'-project: \n\n<b>- <a href="https://www.twitter.com/khgdrn">Kirsten Hagedor n (kaartsamenstelling)</a></b>
```

Comment:
One issue missing:
- `'Deutschlands-BahnhÃ¶fe'` -> `\'Deutschlands-Bahnhöfe\'`

Reply:
Not anymore :wink: